lib/vagrant-ovirt4/action/start_vm.rb

```diff
@@ -46,9 +46,16 @@ def call(env)
           (0...configured_ifaces_options.length()).each do |iface_index|
             iface_options = configured_ifaces_options[iface_index]
 
+            if iface_options[:biosdevname] then
```

Comment:
ignore `biosdevname` altogether and just let the user configure a new param `nic_prefix` imo

Reply:
Biosdevname/systemd has 5 different naming schemes and the interface indexing may from 0, 1 or from 3. I've even seen ens160 in some machines (though not ovirt but vmware). Of course ovirt narrows down the possibilities but still there's options to use virtio/e1000/rtl8139 or even passthough which likely affect the names.

Probably the most reliable and flexible way to cover all the cases would be to leave the whole name for the user to specify and not try to guess the prefixes or how the NICs get indexed. So something like:
:ovirt__ip => '192.168.2.198', ...., :ovirt__nic_name => 'eno2'

Would it be better to close this pull request and open a new one when I get to that point? I'm not too familiar with (or keen on) this github GUI way of doing things.